django_project/geosight/data/api/v1/related_table.py

```diff
@@ -0,0 +1,50 @@
+from drf_yasg.openapi import Response
```

Comment:
Please create file docs like this in the new file (author is your name)
https://github.com/unicef-drp/GeoSight-OS/blob/main/django_project/core/api/access_request.py#L2

Reply:
Done.